dev/automation-hub/initial_data.json

```diff
@@ -174,5 +174,143 @@
       "repository": "787d4137-c9cf-4b26-b66a-aea0e3b6b433",
       "repository_version": null
     }
+  },
+  {
+    "model": "core.remote",
+    "pk": "80ae4b6b-bfb9-4831-aa0c-b43d32ce2fed",
+    "fields": {
+        "name": "Red Hat Certified",
+        "url": "https://cloud.redhat.com/api/automation-hub/v3/collections",
+        "pulp_created": "2020-03-12T15:56:19.071Z",
+        "pulp_last_updated": "2020-03-12T15:56:19.072Z",
+        "pulp_type": "ansible.ansible"
+    }
+  },
+  {
+    "model": "ansible.collectionremote",
+    "pk": "80ae4b6b-bfb9-4831-aa0c-b43d32ce2fed",
+    "fields": {
+        "auth_url": "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
+    }
+  },
+  {
+    "model": "core.repository",
+    "pk": "7eb27e4f-1847-49a2-ba37-a1c3198f923e",
+    "fields": {
+      "pulp_created": "2020-03-12T15:56:19.071Z",
+      "pulp_last_updated": "2020-03-12T15:56:19.072Z",
+      "pulp_type": "ansible.ansible",
+      "name": "rh-certified",
+      "description": "Red Hat Certified",
+      "next_version": 1,
+      "remote": "80ae4b6b-bfb9-4831-aa0c-b43d32ce2fed"
+    }
+  },
+  {
+    "model": "core.repositoryversion",
+    "pk": "a5091fcf-5df0-49b5-8ff0-d84086befec9",
+    "fields": {
+      "pulp_created": "2020-03-12T15:56:19.075Z",
+      "pulp_last_updated": "2020-03-12T15:56:19.075Z",
+      "repository": "7eb27e4f-1847-49a2-ba37-a1c3198f923e",
+      "number": 0,
+      "complete": true,
+      "base_version": null
+    }
+  },
+  {
+    "model": "core.basedistribution",
+    "pk": "a6e40994-3c01-4e0e-8213-a9aebbf463c9",
+    "fields": {
+      "pulp_created": "2020-03-12T15:56:23.436Z",
+      "pulp_last_updated": "2020-03-12T15:56:23.436Z",
+      "pulp_type": "ansible.ansible",
+      "name": "Red Hat Certified",
```

Comment:
we've been using the convention of the distro name, distro base_path and repo name being the same value